gpt_index/indices/query/vector_store/faiss.py

```diff
@@ -53,12 +55,14 @@ def __init__(
         self._faiss_index = faiss_index
 
     def _get_nodes_for_response(
-        self, query_str: str, verbose: bool = False
+        self, query_str: str, verbose: bool = False, **nodes_kwargs: Any
```

Comment:
let's be explicit about keyword arguments (if we need similarity_tracker, which i'm not sure if we need yet).

Reply:
Done.

Reply:
Probably misunderstood how kwargs were being used in the codebase. Makes sense now.